frontend/src/components/LoginForm.vue

```diff
@@ -10,24 +6,32 @@ import { useToast } from 'primevue/usetoast';
 import Toast from 'primevue/toast';
 const router = useRouter();
 const toast = useToast();
+import { hashPassword } from "./password";
 const onFormSubmit = async (event: FormSubmitEvent) => {
     let username = event.states.username.value;
     let password = event.states.password.value;
+    if (!username || !password) {
+      console.log("Empty credentials");
+      toast.add({severity: 'error', summary: 'Credentials cannot be empty', life: 3000});
+      return;
+    }
```

Comment:
Same thing as in `CreateAccountForm.vue`, lines 16-20

Reply:
See Issue #57